### DISCUSSION TITLE
Tell prisma to force ipv4 even if dns return both

### DISCUSSION THREAD

**Discussion by simonep-subbyx** [prisma/prisma#25098] [Q&A]
### Question

Is there any way to tell prism to connect via ipv4? 
When the dns return both ipv4 and ipv6 prisma try to connect with ipv6, but i have db port (3306) open only under ipv4
if in connection string you add connect_timeout=300 for a lot of time prisma try to connect and after a timeout prisma fallback in ipv4

my workaround on ubuntu is add in the end of /etc/gai.conf file:
precedence ::ffff:0:0/96 100

### Expected behavior (optional)

Add an option to force prisma to use ipv4

### Information about Prisma Schema, Client Queries and Environment (optional)

- OS: ubuntu
- Database: MySQL
- Node.js version: 20.15.1
- prisma version: 5.17.0

**Comment by ludralph** [comment]
Hi @simonep-subbyx 👋 

There isn't a direct way to force Prisma to connect via IPv4 when DNS returns both IPv4 and IPv6 addresses. You may want to create a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) and document your use case so that our engineers can review it.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
